src/main/java/ecommerce/dao/impl/jdbc/UserDaoJdbcImpl.java

```diff
@@ -0,0 +1,183 @@
+package ecommerce.dao.impl.jdbc;
+
+import ecommerce.dao.interfaces.UserDao;
+import ecommerce.exceptions.DataProcessingException;
+import ecommerce.lib.Dao;
+import ecommerce.model.Role;
+import ecommerce.model.User;
+import ecommerce.util.ConnectionUtil;
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Statement;
+import java.util.ArrayList;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Optional;
+import java.util.Set;
+
+@Dao
+public class UserDaoJdbcImpl implements UserDao {
+    @Override
+    public Optional<User> findByLogin(String login) {
+        String query = "SELECT * FROM users WHERE login = ?;";
```

Comment:
Why do you need your deleted column in users table if you don't using it. Delete column or change queries